### DISCUSSION TITLE
My account is suspended

### DISCUSSION THREAD

**Discussion by Monneraldo** [community/community#30866] [Other Feature Feedback, Questions, & Ideas] [answered]
So my account was suspended a couple of weeks ago. The reason stated in the email I received was that I was suspended for violating rules about evading permanent suspension. I am sure that this is a mistake as none of my accounts have been suspended previously so I don’t see how I could have evaded permanent suspension from my account. I have sent multiple appeals over the last few days but no response. Even TwitterSupport doesn’t seem to reply to my tweets.
I am aware that this is not the place for queries regarding account suspension but I am really desparate and hence trying out any way possible to get my account back.
Please do let me know if there is any other way I can contact twitter support to get a quicker response.  If you could forward my request to the relevant team, that would be really helpful! My case number is 0284207426

**Comment by deepwateroooMe** [comment]
So you know what, they suspended my deepwaterooo account too, and they do NOT even send an email to me. 
I used my account quite a few years, and have more than >= 40 repositories, and they just suspended it. How urgly they have performed........
They perform so badly that they disputed my pdfs file pages on propose so that my pretty beatifully summaized pdf files can NOT be looked so pretty any more. They are just way too mean........ I guess

**Reply by ExodiaXIV** [reply]
also me too

**Comment by queenofcorgis** [comment, marked as answer]
Hi there!

We're not able to discuss account specifics on here so if you've not already done so please raise a [Reinstatement Request](https://support.github.com/contact/reinstatement)

If you have already raised this please be patient and someone will get back to you as soon as they are able with any information etc. Submitting more requests will only slow any response.

As I say, please exercise patience and someone will be in touch on the email address you fill in on the reinstatement form.

Thanks, 

The Community on Discussions Team
